applications/dashboard/settings/class.hooks.php

```diff
@@ -279,7 +279,8 @@ public function dashboardNavModule_init_handler($sender) {
             ->addGroup(t('Forum Settings'), 'forum', '', ['after' => 'users'])
             ->addGroup(t('Reputation'), 'reputation', '', ['after' => 'forum'])
             ->addGroup(t('Addons'), 'add-ons', '', ['after' => 'reputation'])
-            ->addLinkIf('Garden.Settings.Manage', t('Social Connect'), '/social/manage', 'add-ons.social', '', $sort)
+            ->addLinkIf('Garden.Settings.Manage', t('Social Connect'), '/addons/socialconnect', 'add-ons.social', '', $sort)
+            ->addLinkIf('Garden.Settings.Manage', t('Single Sign On'), '/addons/sso', 'add-ons.sso', '', $sort)
```

Comment:
It would be nice if we could loop through a list of static addon groups defined in the AddonsController, rather than having to manually add each one of them here.